### PR TITLE
Query Max Retry settings

### PR DESCRIPTION
**Pull request by JustinKol**
[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/1688?feature.someFeatureFlagYouMightNeed=true)

Customers are facing challenges due to portal not having a feature to limit the number of retries in case of 429.

### CODE REVIEW

**Comment by JustinKol** [comment]
The compiling errors are due to the cosmos sdk needing to be updated.

File: src/Common/Constants.ts [hunk 1]
```diff
@@ -145,6 +145,9 @@ export class Queries {
   public static QueryEditorMinHeightRatio: number = 0.1;
   public static QueryEditorMaxHeightRatio: number = 0.4;
   public static readonly DefaultMaxDegreeOfParallelism = 6;
+  public static readonly DefaultRetryAttempts = 9;
+  public static readonly DefaultRetryIntervalInMs = 0;
+  public static readonly DefaultMaxWaitTime = 30;
```
Comment (jawelton74) [inline src/Common/Constants.ts:150]: Can we put "InSeconds" at the end of the variable name ?

File: src/Common/Constants.ts [hunk 2]
```diff
@@ -145,6 +145,9 @@ export class Queries {
   public static QueryEditorMinHeightRatio: number = 0.1;
   public static QueryEditorMaxHeightRatio: number = 0.4;
   public static readonly DefaultMaxDegreeOfParallelism = 6;
+  public static readonly DefaultRetryAttempts = 9;
+  public static readonly DefaultRetryIntervalInMs = 0;
```
Comment (jawelton74) [inline src/Common/Constants.ts:149]: The tooltip for this says that default is 5000 ms ?

Comment (JustinKol) [reply src/Common/Constants.ts:149]: Sorry, saw the default was 0 so changed it. I have corrected the tooltip.

**Review by jawelton74 — DISMISSED** [review]

File: src/Common/CosmosClient.ts [hunk 3]
```diff
@@ -150,6 +151,13 @@ export function client(): Cosmos.CosmosClient {
     tokenProvider,
     userAgentSuffix: "Azure Portal",
     defaultHeaders: _defaultHeaders,
+    connectionPolicy: {
```
[the remark this answers is not in the document]
Comment (JustinKol) [reply src/Common/CosmosClient.ts:154]: These are SDK properties so I am not able to change the name.

**Review by vchske — APPROVED** [review]

**Review by jawelton74 — APPROVED** [review]

**Pull request merged**